rmd2redmine.sh

```diff
@@ -0,0 +1,180 @@
+#! /usr/bin/bash
+
+
+htmlFile=""
+
+while getopts ':l:f:p:' opt; do
+  case $opt in
+    f)
+      htmlFile="$OPTARG"
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      exit 1
+      ;;
+    :)
+      echo "Error: -${OPTARG} requires an argument."
+      exit 1
+      ;;
+  esac
+done
+
+echo "fileName = commandArgs(trailingOnly = T)[1]
+
+if (!require('stringr')) {
+  install.packages('stringr')
+}
+if (!require('xml2')) {
+  install.packages('xml2')
+}
+if (!require('rvest')) {
+  install.packages('rvest')
+}
+if (!require('base64enc')) {
+  install.packages('base64enc')
+}
+
+library(stringr)
+library(xml2)
+library(rvest)
+library(base64enc)
+
+reportName = str_remove(fileName, '\\\\.html')
+
+# Set up helper functions
+df_to_textile = function(x) {
+  result = ''
+  if(Negate(is.null)(colnames(x))){
+    for(i in seq_len(ncol(x))){
+      result = paste0(result, '|_.', colnames(x)[i], ' ')
+    }
+    result = paste0(result, ' |\\n')
+  }
+  for(i in seq_len(nrow(x))){
```

Comment:
Depending where you want the code to go (maybe not even this specific code but other), for loops are not allowed - for example Bioconductor does not allow for them - so here you could use apply instead (or vapply, lapply, ...)